packages/manager/tools/testcafe/pages/auth/login.js

```diff
@@ -35,11 +34,6 @@ export default class AuthLoginPage {
   }
 
   async logout() {
-    await t
-      .navigateTo(this.getLogoutUrl())
-      .expect(
-        XPathSelector('//*[@id="login-form"]/div[1]/form/div[1]').innerText,
-      )
-      .contains('You have logged out successfully');
```

Comment:
why do you remove the expect ? like this we can "validate" that we're correctly disconnected

Reply:
seen together i'll add an other check replacing this one